src/problem/npv_163.rs

```diff
@@ -24,11 +24,9 @@ impl fmt::Display for NewTopLevelPackageShouldBeByNameWithCustomArgument {
             file,
         } = self;
         let relative_package_file = structure::relative_file_for_package(package_name);
-        let call_package_arg = if let Some(path) = call_package_path {
-            format!("./{}", path)
-        } else {
-            "...".into()
-        };
+        let call_package_arg = call_package_path
+            .as_ref()
+            .map_or_else(|| "...".into(), |path| format!("./{path}"));
```

Comment:
This is repeated quite a few times. I'm okay with not going DRY, but repeated code often clues you in that there's a chance to encapsulate some more business logic into the type system.

Definitely out of scope for this PR, but might be worth a follow-up issue to flag this.